plugin/helpers.go

```diff
@@ -29,8 +33,22 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// RegisterCommand registers the given command and callback for the ExecuteCommand helper.
+	RegisterCommand(command *model.Command, callback CommandCallback) error
```

Comment:
I've noticed the common convention for callbacks in Golang is usually referred to as `Handlers`, I would recommend renaming this to 
`RegisterCommand(command *model.Command, handler CommandHandler) error` unless there is a good reason not to.

Few examples:
https://github.com/golang/gofrontend/blob/master/libgo/go/net/http/server.go#L1996
https://github.com/golang/go/blob/master/src/syscall/js/func.go#L59

Reply:
Good for me. I'm a Golang noob :)